doozerlib/cli/__main__.py

```diff
@@ -1648,7 +1648,7 @@ def config_mode(runtime, mode, push, message):
 
 
 @cli.command("config:print", short_help="View config for given images / rpms")
-@click.option("-n", "--name-only", default=False, is_flag=True, multiple=True,
+@click.option("-n", "--name-only", default=False, is_flag=True,
```

Comment:
alternatively, if we wanted to keep `multiple=True`, we can just change the default value:
```suggestion
@click.option("-n", "--name-only", default=[], is_flag=True, multiple=True
```

Reply:
ha! Sid was faster: https://github.com/openshift/doozer/pull/443/files